src/GovUk.Education.ExploreEducationStatistics.Common/Utils/Html/HtmlToTextConverter.cs

```diff
@@ -127,32 +127,38 @@ private void ParseElement(IElement element)
                         ParseEdgeNode(element);
                     }
 
-                    AddSpacing(element);
-
                     break;
                 }
             }
+
+            AddSpacing(element);
         }
 
         private void AddSpacing(IElement element)
         {
             var next = element.NextNonWhitespaceSibling();
 
-            if (next is null)
-            {
-                return;
-            }
-
             if (element.IsBlockType() || next is IElement nextElement && nextElement.IsBlockType())
             {
-                // If last two characters are new line characters, then don't
-                // append any more spacing as this would be excessive.
-                if (_builder[^2] == '\n' && _builder[^1] == '\n')
+                // Get last 4 chars to check their line endings. Windows uses CRLF line endings (\r\n),
+                // so there may be up to 4 chars, instead of 2 for LF (\n) when using Linux / Mac.
+                var currentEnding = _builder.Substring(^4..);
+
+                // If already have two line endings, then don't
+                // append more spacing as this would be excessive.
+                if (currentEnding.EndsWith($"{Environment.NewLine}{Environment.NewLine}"))
                 {
                     return;
                 }
 
-                _builder.AppendLine();
+                // Don't a trailing line ending currently, so it should
```

Comment:
Super important comment I know but there's a slight typo here in `Don't a trailing...` :P

Reply:
Good point! Fixed :wink: